Lib/_pyrepl/simple_interact.py

```diff
@@ -103,11 +102,10 @@ def _more_lines(console: code.InteractiveConsole, unicodetext: str) -> bool:
 
 
 def run_multiline_interactive_console(
-    console: code.InteractiveConsole,
+    console: InteractiveConsole,
     *,
     future_flags: int = 0,
 ) -> None:
-    from .readline import _setup
```

Comment:
Please, don't change this line.

Reply:
Ok, but why? I already have import `_setup` on 34 line

Reply:
In general, the default position is the status quo, you need to justify every change.

Reply:
I removed it because it duplicates imports